src/foam/nanos/auth/User.js

```diff
@@ -218,24 +218,12 @@ foam.CLASS({
       displayWidth: 80,
       width: 100,
       preSet: function(_, val) {
-        return val.toLowerCase();
+        return val.toLowerCase().trim();
```

Comment:
Isn't it better if we add this to 'foam.core.EMail' instead, so that it affects all email addresses, and not just this one instance?

Reply:
Which appears in types.js

Reply:
Yeah I agree. I made a change accordingly and pushed it. Also, I changed the class of email property in signUp.js from String to EMail so that it does not need to use preSet anymore.

Reply:
Yeah I agree. I made a change accordingly and pushed it. Also, I changed the class of email property in signUp.js from String to EMail so that it does not need to use preSet anymore.